### ISSUE TITLE
While Installed alongside "Chopdown Updated", Chopdown won't chop trees down

### ISSUE THREAD

**Issue by TeetoKaziim** [orbwoi/UniversalRemote#27]
Forge: 14.23.4.2707
Universal Remote: 1.12.0.9.10
Chopdown Updated: 1.0.12

Expected behaviour: When breaking the last of the ground level logs from under a tree, the chopdown mod usually would turn the rest of that tree into falling item entities that fall onto the ground. But with universal remote installed, trees stoicly hover in the air, in defiance of our will, our code, and our conception of the physical laws of this world.

Basically entire issue is in the title. 

Edit: https://github.com/Shovinus/ChopDownUpdated/issues/12 , in case the two of you don't hate each other and would like to collaborate in the sussing out of this situation.

Thankyou for your time and consideration.

**Comment by Shovinus** [comment]
You can close this mate
